src/Nethermind/Nethermind.Consensus/Transactions/MinGasPriceTxFilter.cs

```diff
@@ -34,7 +34,8 @@ public AcceptTxResult IsAllowed(Transaction tx, BlockHeader? parentHeader, in UI
             UInt256 premiumPerGas = tx.GasPrice;
             UInt256 baseFeePerGas = UInt256.Zero;
             long blockNumber = (parentHeader?.Number ?? 0) + 1;
-            IReleaseSpec spec = _specProvider.GetSpec(blockNumber);
+            ulong blockTimestamp = (parentHeader?.Timestamp ?? 0) + 13;
```

Comment:
Please, please add a comment and fix it ASAP. A separate PR is okay. MinGasPriceTxFilter is used mainly for gnosis when slot = 5s

Reply:
Already on it